insights/tests/__init__.py

```diff
@@ -32,10 +32,10 @@
 
 def _intercept_add_filter(func):
     @wraps(func)
-    def inner(ds, pattern):
-        ret = add_filter(ds, pattern)
+    def inner(component, pattern):
```

Comment:
I've changed `ds` to `component` here, if you think that might be an issue I can change that? But after looking around some, I didn't see any explicit references to `ds`.